python/mxnet/numpy/multiarray.py

```diff
@@ -198,9 +198,9 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):  # pylint: disable=
             kwargs['out'] = out[0]
 
         if method == '__call__':
-            if ufunc.signature is not None:
+            #if ufunc.signature is not None:
```

Comment:
is this un-used code? If this part of code is no longer needed then please remove it.